salt/elife-bot/strip-coverletter.sls

```diff
@@ -58,3 +39,19 @@ strip-coverletter-working:
             ./strip-coverletter.sh /opt/strip-coverletter/dummy.pdf tmp.pdf && rm tmp.pdf
         - require:
             - install-strip-coverletter
+
+#
+# dockerised version
+#
+
+strip-coverletter-docker-image:
+    cmd.run:
+        - cwd: /opt/strip-coverletter
+        - name: ./build-image.sh
```

Comment:
fine for now, but we shouldn't build the image every time, only pulling it from Docker Hub. Can be added later

Reply:
may want to require `docker-ready` in these steps

Reply:
I agree. It is very quick after first build however.

Reply:
done